clients/include/testing_getrf_npvt.hpp

```diff
@@ -70,28 +70,26 @@ hipblasStatus_t testing_getrf_npvt(const Arguments& argus)
     CHECK_HIP_ERROR(hipMemcpy(dA, hA.data(), A_size * sizeof(T), hipMemcpyHostToDevice));
     CHECK_HIP_ERROR(hipMemset(dInfo, 0, sizeof(int)));
 
-    /* =====================================================================
-           HIPBLAS
-    =================================================================== */
-
-    status = hipblasGetrfFn(handle, N, dA, lda, nullptr, dInfo);
-
-    if(status != HIPBLAS_STATUS_SUCCESS)
+    if(argus.unit_check)
     {
-        hipblasDestroy(handle);
-        return status;
-    }
+        /* =====================================================================
+            HIPBLAS
+        =================================================================== */
+        status = hipblasGetrfFn(handle, N, dA, lda, nullptr, dInfo);
 
-    // Copy output from device to CPU
-    CHECK_HIP_ERROR(hipMemcpy(hA1.data(), dA, A_size * sizeof(T), hipMemcpyDeviceToHost));
-    CHECK_HIP_ERROR(hipMemcpy(hInfo1.data(), dInfo, sizeof(int), hipMemcpyDeviceToHost));
+        if(status != HIPBLAS_STATUS_SUCCESS)
+        {
+            hipblasDestroy(handle);
+            return status;
+        }
```

Comment:
Can the localHandle pattern not be used?

Reply:
Yeah, the solver _npvt tests are new so I missed them when doing the other solver tests. I'll add them to hipblas-bench, update the localHandle, etc. in another PR.